<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainmenu</class>
 <widget class="QWidget" name="mainmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="newGameButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highScoresButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>High Scores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Simon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
